examples/rust/components/http-blobstore/wadm.yaml

```diff
@@ -79,14 +79,14 @@ spec:
                     address: 0.0.0.0:8000
 
     # Capability provider that exposes a blobstore with the filesystem
-    - name: blobstore-fs
+    - name: blobstore-nats
       type: capability
       properties:
         # To use a locally compiled provider, uncomment the line below
         # (and ensure that you've used `wash par crate` to create the par file below)
         #
-        # image: file://../../../../crates/provider-blobstore-fs/provider-blobstore-fs.par.gz
-        image: ghcr.io/wasmcloud/blobstore-fs:0.10.2
+        # image: file://../../../../src/bin/blobstore-nats-provider/build/blobstore-nats-provider.par.gz
+        image: ghcr.io/wasmcloud/blobstore-nats:0.1.0
```

Comment:
note to self: once this merges we should release this to ensure the example keeps working 😄

Reply:
@brooksmtownsend, reminding you of the reminder 😂